data/processor.py

```diff
@@ -469,11 +469,11 @@ def _opendata_to_section_meeting(data, term_year):
                 date['weekdays'].replace('U', 'Su'))
 
     # TODO(david): Actually use the term begin/end dates when we get nulls
-    date_format = '%m/%d'
-    start_date = datetime.strptime(date['start_date'], date_format).replace(
-            year=term_year) if date['start_date'] else None
-    end_date = datetime.strptime(date['end_date'], date_format).replace(
-            year=term_year) if date['end_date'] else None
+    date_format = '%m/%d/%Y'
+    start_date = datetime.strptime(date['start_date'] + '/' + str(term_year),
+        date_format).replace(year=term_year) if date['start_date'] else None
```

Comment:
A few questions/comments:

1. How was this working before? Why did things suddenly break?
1. Is the `replace(year=term_year)` part still necessary, now that we're parsing year?
1. I know this isn't your code, but can we separate this into multiple lines? Firstly, I think it's a bad idea to have [hidden] trailing conditionals following a long line. And secondly, I think we have too much going on in one line. E.g. we can construct the date string first, and then pass it to `strptime` in a separate line (because too many characters!)